sb1-scope.json

```diff
@@ -0,0 +1,161 @@
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/barn\\/rest\\/*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/betaling\\/rest\\/payments\\/straksbetaling",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/betaling\\/rest\\/overforing-fra-kredittkort",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/application*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/limit*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/transactions",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/debit",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/debit\\/*order*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/*\\/fraud",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/*\\/pin",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/splitt\\/*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste\\/ask",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste\\/ask",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/utsett\\/betaling*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        }
+      ],
+      "include": [
```

Comment:
host for include will be changed to www.sparebank1.no and mobilbank-pm.sparebank1.no.